### ISSUE TITLE
What is the Linear Regulator for?

### ISSUE THREAD

**Issue by mlolson8** [Herring-UGAECSE-2920/Class-Instructions#3]
The way I see it is that the Motor Driver Hat is to be used to power the X and Y stepper motors and the H-bridge chip is to power the 'Z' Axis motor to raise/lower the pen. And I understand that the 12V to 5V Linear Regulator is to be used in conjunction with the H-Bridge chip, but to do what? In the datasheet for the H-Bridge, it says that it needs 5V for logic switching and 4.5-36V for the driver. I'd assume that we can just power the logic switching voltage directly from the pi, so are we limiting the Output Driver voltage to 5V? Thanks!

**Comment by jealford** [comment]
@mlolson8 This is a great question, we'll actually discuss this in class on Friday.

> are we limiting the Output Driver voltage to 5V?

Short answer, yes. 

Vcc2 is the power supply for the motor and it should be given the rated voltage of the motor. When first testing this H-bridge we assumed (the specs for this commercial plotter are sparse) the z-axis stepper was 12V and so supplied 12V to pin8. This caused the motor to hum and we eventually fried the driver. We've inferred that the stepper runs on 5V because supplying 5V caused the motor to functional properly, hence the regulator.
